File: projects/m1/003-making-change/js/makingChange.js
// toonies = 200 - loonies = 100 - quarters = 25 - dimes = 10 - nickels = 5 - penny = 1

let moneyCashIn,
  moneyCashInConv,
  restMoney,
  toonies,
  loonies,
  half,
  quarters,
  dimes,
  nickels,
  penny;

moneyCashIn = +prompt("Please insert the money you wish to change:");
moneyCashInConv = moneyCashIn * 100;

toonies = Math.floor(moneyCashInConv / 200);
restMoney = moneyCashInConv % 200;

loonies = Math.floor(restMoney / 100);
restMoney = restMoney % 100;

quarters = Math.floor(restMoney / 25);
restMoney = restMoney % 25;

dimes = Math.floor(restMoney / 10);
restMoney = restMoney % 10;

nickels = Math.floor(restMoney / 5);
restMoney = restMoney % 5;

penny = Math.floor(restMoney / 1);
restMoney = restMoney % 1;

alert(`
    To change Money ${moneyCashIn}
    Money covert  in the number integer is ${moneyCashInConv} \n
    Number of Toonies  200 --> ${toonies}
    Number of Loonies 100 --> ${loonies}
    Number of Quarters 25 --> ${quarters}
    Number of Dimes 10 --> ${dimes}
    Number of Nikels 5 --> ${nickels}
    Number of Penny 1 --> ${penny} `);

File: projects/m1/003-making-change/solutions/RoccoFigliamonti/making-change.py
"""# MAKING CHANGE

Consider the software that runs on a self-checkout machine. 
One task that it must be able to perform is to determine how much change to provide when the shopper pays for a purchase with cash.
Write a program that begins by reading a number of cents from the user as an integer. 
Then your program should compute and display the denominations of the coins that should be used to give that amount of change to the shopper. 
The change should be given using as few coins as possible. Assume that the machine is loaded with pennies, nickels, dimes, quarters, loonies and toonies.

# Documentation

For this project solution you may use:

- Variables, expressions, statements
- Strings

# Deadline

This project requires to be completed in a maximum of **2 hours**
"""

"""
pennies= 1 cent
nickels= 5 cent
dimes= 10 cent
quarters= 25 cent
loonies= 1dollar
toonies= 2dollar
"""

price = int(input("total cost of the purchase in cents: "))
payment = int(input("total money paid in cents: "))

change = payment - price

toonies = change//200
loonies = (change-toonies*200)//100
quarters = (change-toonies*200-loonies*100)//25
dimes = (change-toonies*200-loonies*100-quarters*25)//10
nickels= (change-toonies*200-loonies*100-quarters*25-dimes*10)//5
pennies= (change-toonies*200-loonies*100-quarters*25-dimes*10-nickels*5)

moneyname = {"toonies: ": toonies, "loonies: ":loonies, "quarters: ":quarters, "dimes: ":dimes, "nickels: ":nickels, "pennies: ":pennies}

for key in moneyname:
    print(str(key)+str(moneyname[key]))


